exercises/beer-song/src/test/scala/BeerSongTest.scala

```diff
@@ -0,0 +1,60 @@
+import org.scalatest._
+
+class BeerSongTest extends FunSuite {
+
+  test("first generic verse") {
+    val expected = "99 bottles of beer on the wall, 99 bottles of beer.\nTake one down and pass it around, 98 bottles of beer on the wall.\n"
+    val beerSong = new BeerSong()
+    assert(expected == beerSong.verse(99))
+  }
+
+  test("last generic verse") {
+    pending // remove to run test
```

Comment:
Isn't _first generic verse_ the first test?

Before start coding the test suite I checked the [common](https://github.com/exercism/x-common/blob/master/exercises/beer-song/canonical-data.json) and [ruby](https://github.com/exercism/xruby/blob/master/exercises/beer-song/beer_song_test.rb) reference and it seemed to me that it was the first.